content/docs/started/workstation/minikf-gcp.md

```diff
@@ -7,7 +7,7 @@ weight = 4
 This guide describes how to launch a MiniKF virtual machine (VM) on GCP. MiniKF
 is a single VM solution on the Google Cloud Marketplace, and installs:
 
-- Kubernetes (using Minikube)
+- Kubernetes
```

Comment:
We should keep all the mentions of MiniKube on this page, as they related to the implementation of MiniKF, which uses MiniKube under the hood.

Reply:
Oops..forget about this. I have manually reverted the change in minikf-gcp.md.  Please review/approve. Feel free to let me know if further change is needed. 

Thanks @sarahmaddox

Reply:
Thanks!